docs/deployment/under_the_hood.md

```diff
@@ -1,4 +1,4 @@
-# Wagtail Deployment: Under the Hood
+# Wagtail Deployment: under the Hood
```

Comment:
```suggestion
# Deployment: Under the hood
```

This title is not ideal but for now let's just fix the capitalisation and remove the unnecessary Wagtail name.

The word after the colon should probably be capitalised.